app/models/converters/date_time_converter.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+class DateTimeConverter < BaseConverter
+  def self.convert(value)
+    value = super(value)
+
+    return nil if value.blank?
+
+    begin
+      DateTime.parse(value)
```

Comment:
is there a reason this is not `Time.zone.parse(value)`?

Reply:
I don't think we need this converter at all =)